---
title: "Using functional genomic data in monocytes/macrophages and genotyping to nominate disease-driving single nucleotide polymorphisms in juvenile idiopathic arthritis."
collection: publications
permalink: /publication/2024_monocyte_jia
date: 2024-08-19
venue: 'BioRxiv'
paperurl: 'https://www.biorxiv.org/content/10.1101/2024.08.19.608312'
citation: 'E. K. Haley, G. Barshad, <strong>A. Y. He</strong>, E. J. Rice, M. Sudman, S. D. Thompson, E. A. Crinzi,  K. Jiang, C. G. Danko,  and J. N. Jarvis. Using functional genomic data in monocytes/macrophages and genotyping to nominate disease-driving single nucleotide polymorphisms in juvenile idiopathic arthritis. <em>BioRxiv</em>. doi:10.1101/2024.08.19.608312'
---

<strong>Introduction</strong> GWAS have identified multiple regions that confer risk for juvenile idiopathic arthritis (JIA). However, identifying the single nucleotide polymorphisms (SNPs) that drive disease risk is impeded by the SNPsâ€™ that identify risk loci being in linkage disequilibrium (LD) with hundreds of other SNPs. Since the causal SNPs remain unknown, it is difficult to identify target genes and use genetic information to inform patient care. We used genotyping and functional data in primary human monocytes/macrophages to nominate disease-driving SNPs on JIA risk haplotypes and identify their likely target genes.

<strong>Methods</strong> We identified JIA risk haplotypes using Immunochip data from Hinks et al (Nature Gen 2013) and the meta-analysis from McIntosh et al (Arthritis Rheum 2017). We used genotyping data from 3,939 children with JIA and 14,412 healthy controls to identify SNPs that: (1) were situated within open chromatin in multiple immune cell types and (2) were more common in children with JIA than the controls (p< 0.05). We intersected the chosen SNPs (n=846) with regions of bi-directional transcription initiation characteristic of non-coding regulatory regions detected using dREG to analyze GRO-seq data. Finally, we used MicroC data to identify gene promoters interacting with the regulatory regions harboring the candidate causal SNPs.

<strong>Results</strong> We identified 190 SNPs that overlap with dREG peaks in monocytes and126 SNPs that overlap with dREG peaks in macrophages. Of these SNPs, 101 were situated within dREG peaks in both monocytes and macrophages, suggesting that these SNPs exert their effects independent of the cellular activation state. MicroC data in monocytes identified 20 genes/transcripts whose promoters interact with the enhancers harboring the SNPs of interest.

<strong>Conclusion</strong> SNPs in JIA risk regions that are candidate causal variants can be further screened using functional data such as GRO-seq. This process identifies a finite number of candidate causal SNPs, the majority of which are likely to exert their biological effects independent of cellular activation state in monocytes. Three-dimensional chromatin data generated with MicroC identifies genes likely to be influenced by these SNPs. These studies demonstrate the importance of investigations into the role of innate immunity in JIA.